app/assets/stylesheets/admin.scss

```diff
@@ -118,10 +118,34 @@ label {
     padding-top: var(--su-4);
     padding-bottom: var(--su-4);
   }
+
+  &--align {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
```

Comment:
ICYMI, you can also use the utility classes here as well! ([here's what you could use](https://storybook.forem.com/?path=/story/5-css-utility-classes-align-items--items-baseline) instead of `align-items: baseline;`